circulating_supply/src/main.rs

```diff
@@ -50,20 +65,25 @@ pub(super) async fn run_circulating_supply_computation(
             .expect("Time went backwards");
 
         if now < day_to_compute {
+            info!(
+                target: crate::CIRCULATING_SUPPLY,
+                "Waiting until {} to resume calculations",
+                NaiveDateTime::from_timestamp(day_to_compute.as_secs() as i64, 0)
+            );
             tokio::time::sleep_until(tokio::time::Instant::now().add(day_to_compute.sub(now)))
                 .await;
         }
-        wait_for_loading_needed_blocks(&view_client, &day_to_compute).await;
+        wait_for_loading_needed_blocks(&rpc_client, &day_to_compute).await;
 
-        match check_and_collect_daily_circulating_supply(&view_client, &pool, &day_to_compute).await
+        match check_and_collect_daily_circulating_supply(&rpc_client, &pool, &day_to_compute).await
         {
             Ok(_) => {
                 day_to_compute = day_to_compute.add(DAY);
             }
             Err(err) => {
                 error!(
-                    target: crate::AGGREGATED,
-                    "Failed to compute circulating supply for {}: {}. Retry in {} hours",
+                    target: crate::CIRCULATING_SUPPLY,
+                    "Failed to compute circulating supply for {}: {:#}. Retry in {} hours",
```

Comment:
Printing `anyhow` error causes as well as outermost error. I found it quite hard to debug without this 😬